projects/canopy/src/lib/table/table-row/table-row.component.ts

```diff
@@ -46,9 +47,31 @@ export class LgTableRowComponent {
 
   @ContentChildren(LgTableCellComponent) bodyCells: QueryList<LgTableCellComponent>;
 
-  @ContentChildren(LgTableHeadCellComponent) headCells: QueryList<
-    LgTableHeadCellComponent
-  >;
+  @ContentChildren(LgTableHeadCellComponent)
+  headCells: QueryList<LgTableHeadCellComponent>;
+
+  private _tableCellElement: LgTableCellComponent;
```

Comment:
I'm slightly confused by the `tableCellElement` whereabouts is it used? It was initially the naming that confused me with `tableCellElement` and `tableCellComponent` but both being of type `LgTableCellComponent`.

Reply:
Apologies that was a leftover bit of code! I have remove it now as unnecessary